src/data_management/rasters/Raster.js

```diff
@@ -636,13 +632,13 @@ class Raster extends Component {
         </div>
 
         <div>
+          {htmlRasterTableHeader}
```

Comment:
I think the header should also be kept inside the Scrollbars, otherwise, the scrollbar will not work for small screen-size (this is a horizontal scrollbar - there is another vertical scrollbar for the raster table)

Reply:
This scrollbar actually also functions as a vertical scrollbar for the whole table (header and body of the table) if the header is also put in the Scrollbar.
I added it within the Scrollbar again and made the overflow-y hidden.